### DISCUSSION TITLE
All models rejected

### DISCUSSION THREAD

**Discussion by qrtise** [TheLastBen/fast-stable-diffusion#2745] [Q&A]
I got the collab working fine the first couple of goes, however now no matter which model I use it is rejected. This is the error code I get... any help is much appreciated. 

To: /content/model.safetensors
100% 2.63G/2.63G [02:35<00:00, 16.9MB/s]
Detecting model version...
---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
[<ipython-input-6-1b40d2cfa913>](https://localhost:8080/#) in <cell line: 149>()
    282         wget.download('https://github.com/TheLastBen/fast-stable-diffusion/raw/main/Dreambooth/det.py')
    283         print('Detecting model version...')
--> 284         Custom_Model_Version=check_output('python det.py '+sftnsr+' --MODEL_PATH '+modelnm, shell=True).decode('utf-8').replace('\n', '')
    285         clear_output()
    286         print(''+Custom_Model_Version+' Detected')

1 frames
[/usr/lib/python3.10/subprocess.py](https://localhost:8080/#) in run(input, capture_output, timeout, check, *popenargs, **kwargs)
    524         retcode = process.poll()
    525         if check and retcode:
--> 526             raise CalledProcessError(retcode, process.args,
    527                                      output=stdout, stderr=stderr)
    528     return CompletedProcess(process.args, retcode, stdout, stderr)

CalledProcessError: Command 'python det.py --from_safetensors --MODEL_PATH model.safetensors' returned non-zero exit status 1.

**Comment by TheLastBen** [comment]
check if the link is downloadable

**Reply by qrtise** [reply]
The links were downloadable. I tried using a different account and Google Drive. The collab worked fine. 
I next downloaded my entire Gdrive on the original account and then deleted the contents. 
I then used your collab and it worked perfectly. I am not sure why this happened... Maybe some residual code I didn't delete from my last use.
